src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -145,4 +161,33 @@ public void stop(StopConfiguration configuration) throws AppEngineException {
       }
     }
   }
+
+  @VisibleForTesting
+  void checkAndWarnIgnored(Object valueToIgnore, String propertyName) {
+    if (valueToIgnore != null) {
+      log.warning(propertyName + " will be ignored by Dev Appserver v1");
+    }
+  }
+
+  @VisibleForTesting
+  String determineJavaRuntimeVersion(List<File> services) {
```

Comment:
It's actually testing more than one service to determine what runtime we should assume. I think I should add a comment on top of this.